src/goto-symex/path_storage.h

```diff
@@ -90,6 +92,8 @@ class path_storaget
   /// Counter for nondet objects, which require unique names
   symex_nondet_generatort build_symex_nondet;
 
+  std::unordered_map<irep_idt, local_safe_pointerst> safe_pointers;
```

Comment:
:pick: You might choose to doxygen this variable, `safe_pointers` doesn't mean mich to me, but perhaps just a reference to a definition of a "safe pointer" would be a good addition here.

Reply:
Documentation added.